test/Test/Core/Issue.hs

```diff
@@ -0,0 +1,52 @@
+module Test.Core.Issue where
+
+import IW.Core.Id (Id (..))
+import IW.Core.Issue (Issue (..))
+import IW.Core.SqlArray (SqlArray (..))
+
+import Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+testLabels :: [Text]
+testLabels = [ "good first issue"
+             , "help wanted"
+             , "low hanging fruit"
+             , "docs"
+             , "easy"
+             ]
+
+genIssue :: MonadGen m => m Issue
+genIssue = do
+    id' <- genId
+    number <- genNumber
+    title <- genTitle
+    body <- genBody
+    repoOwner <- genRepoOwner
+    repoName <- genRepoName
+    labels <- genLabels
+    -- | Construct issue url
+    let url = "https://github.com/" <> repoOwner <> "/" <> repoName <> "/issues/" <> show number
+
+    pure $ Issue id' number title body url repoOwner repoName labels 
+    where
+        genId :: MonadGen m => m (Id Issue)
```

Comment:
The indentation here is wrong. It should be 2 spaces before `where` and 4 spaces before functions inside `where`